src/Symfony/Component/Notifier/Receiver/AdminReceiver.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Receiver;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+class AdminReceiver extends Receiver implements SmsReceiverInterface
+{
+    private $phone = '';
```

Comment:
The assigment can be safely removed as the constructor overrides it

Reply:
removed